packages/flutterfire_cli/lib/src/commands/upload_symbols.dart

```diff
@@ -313,7 +313,10 @@ class UploadCrashlyticsSymbols extends FlutterFireCommand {
     }
 
     if (projectId == null || appId == null) {
-      throw FirebaseJsonException();
+      final underlyingException =
```

Comment:
I was also thinking about adding a separate message for a case when both `projectId` and `appId` is missing, but I'm not sure if it's needed - worst case user will first see `projectId is missing` and then `appId is missing`

Reply:
Yes, I think this works 😄